website/docs/d/private_dns_glb_monitors.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "ibm"
+page_title: "IBM : "
+sidebar_current: "docs-ibm-datasources-dns-glb-monitors"
+description: |-
+  Manages IBM Cloud Infrastructure Private Domain Name Service GLB monitors.
+---
+
+# ibm\_dns_glb_monitors
+
+Import the details of an existing IBM Cloud Infrastructure private domain name service GLB monitors as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+
+## Example Usage
+
+```hcl
+
+data "ibm_dns_glb_monitors" "ds_pdns_glb_monitors" {
+  instance_id = "resource_instance_guid"
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `instance_id` - (Required, string) The resource instance id of the private DNS on which zones were created.
+* `zone_id` - (Required, string) The unique identifier of the private DNS zone.
```

Comment:
I don't see any argument for ibm_dns_monitor datasource. is zone_id needed here?

Reply:
i removed Zone_id its not required